package.json

```diff
@@ -21,7 +21,7 @@
     "lint": "dtslint types"
   },
   "devDependencies": {
-    "dtslint": "Microsoft/dtslint#production",
+    "dtslint": "github:Microsoft/dtslint#production",
```

Comment:
Why is this needed?

Reply:
I mentioned above, any changes to the global `package.json` should go in their own PRs.